services/dumper.js

```diff
@@ -174,6 +174,44 @@ function Dumper(project, config) {
     fs.writeFileSync(`${pathDest}/models/index.js`, text);
   }
 
+  function writeDockerfile(pathDest) {
+    const templatePath = `${__dirname}/../templates/app/Dockerfile`;
+    const template = _.template(fs.readFileSync(templatePath, 'utf-8'));
+
+    const settings = {
+      port: config.appPort,
+    };
+
+    fs.writeFileSync(`${pathDest}/Dockerfile`, template(settings));
+  }
+
+  function writeDockerCompose(pathDest, authSecret) {
+    const templatePath = `${__dirname}/../templates/app/docker-compose.yaml`;
+    const template = _.template(fs.readFileSync(templatePath, 'utf-8'));
+
+    const settings = {
+      appName: config.appName,
+      forestEnvSecret: project.defaultEnvironment.secretKey,
+      forestAuthSecret: authSecret,
+      databaseUrl: getDatabaseUrl(),
+      forestUrl: process.env.FOREST_URL,
+      devRenderingId: project.defaultEnvironment.renderings[0].id,
+      ssl: config.ssl,
+      encrypt: config.ssl && config.dbDialect === 'mssql',
+      dbSchema: config.dbSchema,
+      port: config.appPort,
+    };
+
+    fs.writeFileSync(`${pathDest}/docker-compose.yaml`, template(settings));
```

Comment:
It's `docker-compose.yml`, not `yaml`

Reply:
From the doc it seems we can use both:

https://docs.docker.com/compose/compose-file/#service-configuration-reference

`Tip: You can use either a .yml or .yaml extension for this file. They both work.`